test/model-serialization.test.js

```diff
@@ -16,8 +16,12 @@ function getImageData (src, callback) {
 }
 
 function initFunc (plates) {
-  // const bluePlate = plates[210] // 210 hue
+  const bluePlate = plates[210] // 210 hue
   const yellowPlate = plates[70] // 70 hue
+  const purplePlate = plates[300] // 300 hue
```

Comment:
Updating the densities here did correct the broken test...even though I'm still not sure why the test broke in the first place